modules/lvms-about-volume-snapshots.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="lvms-about-volume-snapsot_{context}"]
+= About volume snapshots
+
+You can create snapshots of persistent volume claims (PVCs) that are provisioned by {lvms}.
+
+You can perform the following actions using the volume snapshots:
+
+* Back up your application data.
++
+[IMPORTANT]
+====
+Volume snapshots are located on the same devices as the original data. To use the volume snapshots as backups, you must move the snapshots to a secure location. You can use OpenShift API for data protection backup and restore solutions.
+====
+
+* Revert to a state at which the volume snapshot was taken.
+
+[NOTE]
+====
+You can also create volume snapshots of the volume clones. 
+====
+
+== Limitations for creating volume snapshots in multi-node topology
+
+{lvms} has the following limitations for creating volume snapshots in multi-node topology:
+
+* Creating volume snapshots is based on the LVM thin pool capabilities.
+* The node must have additional storage after creating a volume snapshot for further updating the original data source.
+* You can create volume snapshots only on the node where you have deployed the original data source.
+* Pods relying on the PVC that uses the snapshot data can be scheduled only on the node where you have deployed the original data source.
+* If the PVC is not present and available before creating a volume snapshot, the scheduler in multi-node {product-title} clusters does not provision the volume snapshots on the node where you have deployed the original data source.
```

Comment:
@jakobmoellerdev Could you take a look if above scenario/statement is valid?

Reply:
> The node must have additional storage after creating a volume snapshot for further updating the original data source.

I dont know what "additional storage" means here, we should change the wording here to be more concrete.

> * If the PVC is not present and available before creating a volume snapshot, the scheduler in multi-node {product-title} clusters does not provision the volume snapshots on the node where you have deployed the original data source.

I suggest to change this to 

> * If the PVC is not `Bound` before creating a volume snapshot, the scheduler in multi-node {product-title} clusters might not provision the volume snapshots on the node where you have deployed the original data source, causing the snapshot to fail.

Reply:
Not sure about this.. need to think of the wording but thats not quite it. @suleymanakbas91 any ideas?

Reply:
Due to above discussion, lets drop:

If the PVC is not in the Bound state before creating a volume snapshot, the scheduler in multi-node {product-title} clusters does not provision the volume snapshots on the node where you have deployed the original data source.

Reply:
Removed this point.